{"name": "Проверка авторизации с неверным паролем", "status": "passed", "description": "Создаем пользователя, авторизуемся с неверным паролем,                проверяем, что код ответа: 401, и текст ответа \"email or password are incorrect\"", "start": 1713017275250, "stop": 1713017276352, "uuid": "4fa5d6cb-3016-422b-a2ee-e79a7aa9887f", "historyId": "53bfe258a014bcf7ff35c00adc19c24f", "testCaseId": "53bfe258a014bcf7ff35c00adc19c24f", "fullName": "tests.test_auth_user.TestAuthUser#test_login_user_unvalid_password_faild", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestAuthUser"}, {"name": "host", "value": "DESKTOP-1VNC7DB"}, {"name": "thread", "value": "12296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}